### ISSUE TITLE
Git Blame in the Powerline

### ISSUE THREAD

**Issue by idoo** [TheBB/spaceline#118]
I know that we have a magit with blame functions, however I suppose that a git blame for current line in powerline is very useful feature. Do you have any ideas how I can add it? and may be it's good idea to add this snippet to documentation?

**Comment by TheBB** [comment]
> Do you have any ideas how I can add it?

Yes the readme describes how to write a segment. The file `spaceline-segments.el` is full of examples.

> and may be it's good idea to add this snippet to documentation?

Or the code…

**Comment by idoo** [comment]
I tried add new snippet to powerline with

```
(spaceline-define-segment hello
  "Say hello"
  "hello")
```
and activated it though `spaceline-hello-on`, but it's doesn't showed on `spaceline`

could somebody please explain me, what I missed?

**Comment by TheBB** [comment]
Did you put it in the modeline?

**Comment by idoo** [comment]
@TheBB no, to my emacs config :( So, I suppose it's wrong
